testem.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-  test_page: 'tests/index.html?hidepassed',
+  test_page: 'tests/index.html?hidepassed&nolint',
```

Comment:
I added `nolint` query parameter so that `ember-qunit` skips lint checks by `ember-cli-dependency-lint`.